src/Installer/core-sdk-tasks/GenerateRuntimeAnalyzersSWR.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class GenerateRuntimeAnalyzersSWR : Task
+    {
+        [Required]
+        public string RuntimeAnalyzersLayoutDirectory { get; set; }
+
+        [Required]
+        public string OutputFile { get; set; }
+
+        public override bool Execute()
+        {
+            StringBuilder sb = new StringBuilder(SWR_HEADER);
+
+            AddFolder(sb,
+                      @"AspNetCoreAnalyzers",
+                      @"SDK\AspNetCoreAnalyzers");
```

Comment:
Should the target VS folder be something like `DotNetSdk` rather than just `SDK`?